RIRImageResizing/Classes/NSObject/RIRResizeImageOperationParameters.h

```diff
@@ -0,0 +1,43 @@
+//
+//  RIRResizeImageOperationParameters.h
+//  Pods
+//
+//  Created by Richard Reitzfeld on 11/21/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+
+
+
+
+typedef NS_ENUM(NSInteger, UIImage_RIRResizing_ResizeMode) {
```

Comment:
Go ahead and

1. Put these `UIImage_RIRResizing_ResizeMode` in their own file, `UIImage_RIRResizing_ResizeModes.h`.
2. Add a `UIImage_RIRResizing_ResizeMode_none` before the rest of them (so it's value 0). Don't update first or last, we want this one to be out of range.